components/clearbutton/index.css

```diff
@@ -10,99 +10,101 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    --spectrum-clear-button-width: var(--spectrum-component-height-300);
   }
-}
 
-.spectrum-ClearButton--sizeM {
-  @remapvars {
-    find: --spectrum-clearbutton-m-;
-    replace: --spectrum-clearbutton-;
+  &:disabled {
+    --mod-clear-button-icon-color: var(--spectrum-clear-button-icon-color-disabled, var(--spectrum-disabled-content-color));
+    --mod-clear-button-icon-color-hover: var(--spectrum-clear-button-icon-color-hover-disabled, var(--spectrum-disabled-content-color));
+    --mod-clear-button-icon-color-down: var(--spectrum-clear-button-icon-color-down-disabled, var(--spectrum-disabled-content-color));
+    --mod-clear-button-background-color: var(--spectrum-clear-button-background-color-disabled, var(--spectrum-disabled-background-color));
   }
-}
 
-.spectrum-ClearButton--sizeL {
-  @remapvars {
-    find: --spectrum-clearbutton-l-;
-    replace: --spectrum-clearbutton-;
+  &.spectrum-ClearButton--quiet {
+    --mod-clear-button-background-color: var(--spectrum-clear-button-background-color-quiet, transparent);
+    --mod-clear-button-background-color-hover: var(--spectrum-clear-button-background-color-hover-quiet, transparent);
+    --mod-clear-button-background-color-down: var(--spectrum-clear-button-background-color-down-quiet, transparent);
+    --mod-clear-button-background-color-key-focus: var(--spectrum-clear-button-background-color-key-focus-quiet, transparent);
   }
-}
 
-.spectrum-ClearButton--sizeXL {
-  @remapvars {
-    find: --spectrum-clearbutton-xl-;
-    replace: --spectrum-clearbutton-;
+  &.spectrum-ClearButton--overBackground {
+    --mod-clear-button-icon-color: var(--spectrum-clear-button-icon-color-over-background, var(--spectrum-white));
+    --mod-clear-button-icon-color-hover: var(--spectrum-clear-button-icon-color-hover-over-background, var(--spectrum-white));
+    --mod-clear-button-icon-color-down: var(--spectrum-clear-button-icon-color-down-over-background, var(--spectrum-white));
+    --mod-clear-button-icon-color-key-focus: var(--spectrum-clear-button-icon-color-key-focus-over-background, var(--spectrum-white));
+
+    --mod-clear-button-background-color: var(--spectrum-clear-button-background-color-over-background, transparent);
+    --mod-clear-button-background-color-hover: var(--spectrum-clear-button-background-color-hover-over-background, var(--spectrum-transparent-white-300));
+    --mod-clear-button-background-color-down: var(--spectrum-clear-button-background-color-hover-over-background, var(--spectrum-transparent-white-400));
+    --mod-clear-button-background-color-key-focus: var(--spectrum-clear-button-background-color-hover-over-background, var(--spectrum-transparent-white-300));
   }
 }
 
-.spectrum-ClearButton-fill {
-  background-color: var(--spectrum-clearbutton-fill-background-color);
-
-  inline-size: var(--spectrum-clearbutton-fill-size);
-  block-size: var(--spectrum-clearbutton-fill-size);
+.spectrum-ClearButton {
```

Comment:
we lost the `cursor: pointer;` with removing `%spectrum-BaseButton` so there is no longer a cursor on hover. I think it would make sense to add back the `cursor: pointer;` here.

Reply:
Good call out!